tests/yam/agama/agama.pm

```diff
@@ -8,25 +8,34 @@
 use base Yam::Agama::agama_base;
 use strict;
 use warnings;
-
+use Carp qw(croak);
 use testapi qw(
   get_required_var
   script_run
   assert_script_run
   record_soft_failure
+  parse_extra_log
+  upload_logs
 );
 
 sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
     my $test_options = get_required_var('AGAMA_TEST_OPTIONS');
     my $reboot_page = $testapi::distri->get_reboot();
+    my $spec = "$test.spec";
+    my $tap = "$test.tap";
+    my $reporters = "--test-reporter=spec --test-reporter=tap --test-reporter-destination=/tmp/$spec --test-reporter-destination=/tmp/$tap";
 
     script_run("dmesg --console-off");
-    assert_script_run("node --enable-source-maps /usr/share/agama/system-tests/" . $test . ".js " .
-          $test_options, timeout => 2400);
+    my $ret = script_run("node --enable-source-maps $reporters /usr/share/agama/system-tests/${test}.js $test_options", timeout => 2400);
```

Comment:
can we show in openQA the content of spec after this command? with `diag` or something like that?